modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -774,7 +785,8 @@ public CompletableFuture<Ignite> start(Path configPath) {
                                     indexManager,
                                     qryEngine,
                                     clientHandlerModule,
-                                    deploymentManager
+                                    deploymentManager,
+                                    systemViewManager // Should be the last one.
```

Comment:
that's cool you put the note to keep view manager as the last one. It would be nice to add a few more words why exactly we have to keep it the last one though

Reply:
Done